tests/test_integration.py

```diff
@@ -160,6 +162,34 @@ def check_vm_absent(self, hv_name=None):
                     )
 
 
+class SettingsHardwareModelTest(TestCase):
+    """Test that all hardware_models of all hypervisors have a cpu threshold
+    value defined in HYPERVISOR_CPU_THRESHOLDS dict"""
+
+    def setUp(self):
+        self.hardware_models = set([
+            x['hardware_model'] for x in Query({
+                'servertype': 'hypervisor',
+                'project': 'ndco',
+                'environment': Any('production', 'testing'),
```

Comment:
Please don't check the environment at all.

Reply:
Sure. Deleting stuff is easy.